src/main/resources/com/sonyericsson/jenkins/plugins/bfa/PluginImpl/config.properties

```
@@ -3,3 +3,4 @@ nrOfScanThreadsDescription=Number of threads per build to use when scanning the
 testResultParsingEnabledDescription=Treat failed test cases (as indicated by JUnit/xUnit/... publishers) as failure causes.
 testResultCategoriesDescription=A space-separated list of categories to use for failure causes representing failed test cases.
 maxLogSize=Log file with size that exceeds limit (in MB) would not be scanned, 0 - disables this check
+fallbackCategoriesDescription=Causes containing on of these (space separated) categories will only be applied if there are no non-fallback causes found.
```

```suggestion
fallbackCategoriesDescription=Space separated list of category names that marks fallback causes. Fallback causes will only be applied if there are no non-fallback causes found.
```

I don't know if my version is better, but I had a really hard time parsing/reading the original one (maybe only because __one__ was missing an _e_) :)